docs/design-system/ui_kits/ovp/index.html

```
@@ -0,0 +1,600 @@
+<meta charset="UTF-8">
+<title>OVP — Local Web UI</title>
+<link rel="stylesheet" href="ovp-ui.css">
+<link rel="icon" type="image/svg+xml" href="../../assets/monogram.svg">
+</head>
+<body>
+
+<div class="page">
+  <div class="screen-tabs">
+    <div class="screen-tabs-inner">
+      <span class="label">UI KIT · 4 SCREENS</span>
+      <button data-screen="reader" class="active">Reader · Home</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)